payloads/library/remote_access/Root_Reverse_Shell_linux_mac/README.md

```diff
@@ -0,0 +1,17 @@
+# Root_Reverse_Shell_linux_mac
+
+### Since i dont have a bash bunny this is tested in digispark
+### I have converted this script to bash bunny
+### If any issues put in discussion i will fix it
+POC DIGISPARK LINK : https://drive.google.com/open?id=1DvKX8QXHImVRZMaoTvmtreFkiL4rwYF-
+### Special thanks to sudobackdoor for bash script sample
+Dont forgot to change IP in payload.sh
+Before using this payload don't forgot to start netcat listeners on port 4444 and 1337

```

Comment:
Before using this payload don't forget to start `netcat` listeners on port 4444 and 1337